settings.py

```diff
@@ -39,6 +39,7 @@
 # the number of seconds chaosbot should sleep between polling for ready prs
 PULL_REQUEST_POLLING_INTERVAL_SECONDS = 30
 ISSUE_COMMENT_POLLING_INTERVAL_SECONDS = 60 * 10  # 10 min window on polling comments
+ISSUE_CLOSE_STALE_INTERVAL_SECONDS = 60 * 10  # 10 min window on polling issues
```

Comment:
@phil-r I agree completely. I wanted to do it every other hour or so. The problem is that if a PR is accepted in the middle of that interval, the timer resets... which means that the cron never runs. :hourglass: